vmware/ngc/NGC-Register/src/main/java/org/opensds/vmware/ngc/utils/AesUtils.java

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package org.opensds.vmware.ngc.utils;
+
+import org.apache.logging.log4j.Logger;
+
+public class AesUtils {
```

Comment:
why do we need this file and class now (not implemented!). Should we echo any message when called?

Reply:
This is a placeholder for now. Will be implementing this in next step.